highcharts/highcharts
# tools/comment-json

See https://github.com/highcharts/highcharts/pull/21817 for comment with visual changes

## Test plan
1. Create a new branch from this branch
2. Introduce a visual change (edit the height of a demo in `samples/highcharts/demo/**`
3. Make a draft PR
4. The comment bot should comment with the expected diff
5. Maybe optionally: revert the change and check that the comment now says `No difference found`

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207023166064806
